SmartCarMQTTApp/app/src/main/java/com/example/smartcarmqttapp/screens/PracticeDrivingActivity.java

```diff
@@ -102,14 +110,14 @@ protected void onCreate(Bundle savedInstanceState) {
         sensorDialog = new Dialog(this);
 
         dashboard();
-        if (CarState.instance.isConnected()) {
-            try {
-                CarState.instance.getConnectedCar().changeSpeed(25.0);
-            }
-            catch(Exception ex) {
-                // ignore
-            }
-        }
+//        if (CarState.instance.isConnected()) {
+//            try {
+//                CarState.instance.getConnectedCar().changeSpeed(25.0);
+//            }
+//            catch(Exception ex) {
+//                // ignore
+//            }
+//        }
```

Comment:
You shouldn't merge this while these lines of code are commented out. Either remove them altogether or fix the issue.
You may use it as it is for discussion and solving the issue of course.

Reply:
Thanks for the feedback! Does this apply to all commented-out code?

Reply:
yes!

Reply:
Yes!

Reply:
Yeap, unless there's a very significant reason there shouldn't be any commented out code committed to a main branch.
Some tools event detect and flag comments containing code as a code smell.